packages/catalog-process/test/db.test.ts

```diff
@@ -724,6 +749,14 @@ describe("database test", async () => {
         });
         expect(writtenPayload.eService).toEqual(toEServiceV1(eService));
         expect(writtenPayload.descriptorId).toEqual(descriptor.id);
+        expect(deleteFile).toHaveBeenCalledWith(
```

Comment:
I added this check to see if the `deleteFile` function is called once for the `interface` and once for the `document` (indeed the docs array contains one document in this case).